FirebaseRemoteConfig/SwiftNew/ConfigDBManager.swift

```diff
@@ -0,0 +1,376 @@
+// Copyright 2024 Google LLC
```

Comment:
This file is the internal API for the DB. The guts of the implementation is in DatabaseActor which is the code that was previously in a dedicated queue.